### ISSUE TITLE
Pre-training

### ISSUE THREAD

**Issue by Xcrid** [lianqing11/PyCDA#4]
Hi,

When you pre-train your model with the source dataset, do you also pre-train the aux_output by using the aux_loss or do you only use the main loss ?

**Comment by lianqing11** [comment]
Hi, 

I use the aux loss and the loss weight for aux loss is 0.4. Actually I try both, with or without aux loss get similar results.  The gap between them is smaller than the randomness in the target domain. You can try the source only with aux loss by modifying train_adabn.py (just delete the target domain part). If you find that I do not reply to you in time, you can send an email to lianqing1997@gmail.com since GitHub does not has the message notification.

Best regards,
Qing LIAN

**Comment by Xcrid** [comment]
Thanks, I'll try that
